packages/selected-network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.2]
+### Uncategorized
+- Fix SelectedNetworkController default metamask networkClientId ([#1757](https://github.com/MetaMask/core/pull/1757))
+- Release 81.0.0 ([#1747](https://github.com/MetaMask/core/pull/1747))
+- devDeps: @metamask/eslint-config*->12.1.0 ([#1740](https://github.com/MetaMask/core/pull/1740))
```

Comment:
A few things:

- According to the changelog notes for PR 1757, its entry should go in "Fixed"
- We see an entry called "Release 81.0.0" because internal dependencies for selected-network-controller were bumped last release but a new release didn't go out for this package (this was intentional, so all is fine). We should replace this with the actual changes, though
- We don't need the `devDeps` entry, since it's development-only.

Putting this together, I think we have:

``` suggestion
### Changed
- Bump dependency on `@metamask/base-controller` to ^3.2.3 ([#1747](https://github.com/MetaMask/core/pull/1747))
- Bump dependency and peer dependency on `@metamask/network-controller` to ^14.0.0 ([#1747](https://github.com/MetaMask/core/pull/1747))

### Fixed
- `setNetworkClientIdForDomain()` will now ignore the passed in domain value and set the `networkClientId` for the metamask domain instead when the `state.perDomainNetwork` flag is false (default) ([#1757](https://github.com/MetaMask/core/pull/1757))